packages/webapp/src/components/Finances/AddExpense/styles.module.scss

```diff
@@ -51,6 +51,7 @@ p.iconLink {
 .type {
   line-height: 20px;
   margin-bottom: 8px;
+  font-weight: bold;
```

Comment:
Just curious if this changes the look for you? I see no change.

Reply:
Where I see this change is on the screenshots at the top of this PR. If you see the first screenshot, the title of the expense type (**Equipment**) has a normal font weight. But in the second screenshot, the title of the expenses (**Equipment** and **Fuel**) have a bold font weight.

Did you use any browser different from Chrome? If so, please let me know which one.

Reply:
Oh I see it -- I was definitely looking in the wrong spot 😂 . Thanks!